test/multivalue.wast

```diff
@@ -53,14 +53,16 @@
  )
 
  ;; Test multivalue globals
- (func $global (result i32 i64)
+ (func $global (result i64)
   (global.set $g1
    (tuple.make
     (i32.const 42)
     (i64.const 7)
    )
   )
-  (global.get $g2)
+  (tuple.extract 1
+   (global.get $g2)
+  )
```

Comment:
Should we also have a test of a `global.get` of a tuple that does not flow into an `extract`?